File: springboot-dubbo-client/src/main/java/cn/liangqinghai/study/spring/dubbo/controller/IndexController.java
package cn.liangqinghai.study.spring.dubbo.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class IndexController {

    @GetMapping({"/", "/one"})
    public String one() {
        return "one";
    }

    @PostMapping("/two")
    public void two() {

        System.out.println("two");

    }

}

File: kotlin-study-springboot/src/main/kotlin/cn/liangqinghai/study/kotlin/springboot/controller/IndexController.kt
package cn.liangqinghai.study.kotlin.springboot.controller

import cn.liangqinghai.study.kotlin.springboot.model.People
import cn.liangqinghai.study.kotlin.springboot.service.PeopleService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class IndexController {

    @Autowired
    val peopleService : PeopleService? = null

    @GetMapping(value = ["/", "index"])
    fun index(): Any {
        return "Hello kotlin"
    }

    @GetMapping("/people")
    fun people(): MutableIterable<People>? {

        return peopleService?.getAllPeople()

    }

}